doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -795,7 +795,13 @@ Modem libraries
 Multiprotocol Service Layer libraries
 -------------------------------------
 
-|no_changes_yet_note|
+* Added integration with nrf2 clock control driver for nRF54h20 SoC
+* Added integration with Zephyr's System Power Management for nRF54h20 SoC
+* Added global domain HSFLL120 320MHz frequency request if MPSL is enabled.
+  The high frequency in global domain is required to ensure that instructions fetching from L2-cache and MRAM is as fast as possible.
+  It is needed for radio protocols to operate correctly.
+* Added MRAM always-on request for scheduled radio events.
+  It is needed to avoid MRAM wake-up latency for radio protocols.
```

Comment:
```suggestion
* Added integration with nrf2 clock control driver for nRF54H20 SoC.
* Added integration with Zephyr's System Power Management for nRF54H20 SoC.
* Added global domain HSFLL120 320MHz frequency request if MPSL is enabled.
  The high frequency in global domain is required to ensure that instructions fetching from L2-cache and MRAM is as fast as possible.
  It is needed for radio protocols to operate correctly.
* Added MRAM always-on request for scheduled radio events.
  It is needed to avoid MRAM wake-up latency for radio protocols.
```

Reply:
done :-)